### PR TITLE
style(app): App style design feedback slideout factory reset

### PR DESCRIPTION
**Pull request by koji**
# Overview
This PR will fix Factory Slideout design feedback and update checkbox field styling.

**In terms of banner styling, I need Mel's help to understand what she pointed out exactly.**

# Changelog
- Update Factory Reset slideout styling
- Add text style to checkboxField

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/Devices/RobotSettings/AdvancedTab/AdvancedTabSlideouts/FactoryResetSlideout.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {
   SPACING,
   TYPOGRAPHY,
   Link,
-  CheckboxField,
+  // CheckboxField,
```
Comment (jerader) [inline app/src/organisms/Devices/RobotSettings/AdvancedTab/AdvancedTabSlideouts/FactoryResetSlideout.tsx:13]: this should be deleted

File: app/src/atoms/CheckboxField/index.tsx [hunk 2]
```diff
@@ -90,6 +90,19 @@ const INNER_STYLE_NO_VALUE = css`
     color: ${COLORS.disabled};
   }
 `
+
+const LABEL_TEXT_STYLE = css`
+  font-size: ${TYPOGRAPHY.fontSizeP};
+  font-weight: ${TYPOGRAPHY.fontWeightRegular};
+  color: ${COLORS.darkBlack};
+  flex: 0 0 auto;
+  padding: 0 0.5rem;
```
Comment (jerader) [inline app/src/atoms/CheckboxField/index.tsx:99]: this can be `0 ${SPACING.spacing3}`;

**Review by jerader — APPROVED** [review]
just a couple of comments but lgtm!

**Comment by mmencarelli** [comment]
Round 2 feedback on factory reset slideout 

- [x] (1) Up the size of this icon to 24px, currently it's 20px 
- [ ] (2) Wondering why this is a different style from design? 
- [ ] (3) Add 8px margin between the list items
- [ ] (4) Looks like there's a barely noticeable grey line right underneath the button? 
- [ ] (5) remove box shadow from `.layXKe`
- [ ] (6) Looks like the SVG is a bit thicker than the designs—when you get to this step, contact me and I can try sending you another SVG
- [ ] (7) The hover state colour should be our darkGreyHover (#646468), currently it's #283d52
- [ ] (8) When I try to tab through all the links using my keyboard none of the links of the tabs go into their focusVisible state


Screen Shot 2022-06-21 at 3 25 24 PM

**Pull request merged**